modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -387,6 +388,21 @@ const AddFDSEntryModalContent = ({
 		}
 	};
 
+	const validatePath = (path: string, allowedParameters: string[]):boolean => {
+		const paramsMatcher = RegExp('\{(.*?)\}', 'g');
+		let isValid = true;
+		let matches;
+	
+		while ((matches = paramsMatcher.exec(path)) !== null) {
+			if (! allowedParameters.includes(matches[1])){
+				isValid = false;
+				break;
+			}
+		}
+
+		return isValid;
+	}	
+
```

Comment:
We can simplify this, by leveraging `some()`. Something like:
```js
if (path.includes('{') && !ALLOWED_ENDPOINTS_PARAMETERS.some((param) => path.includes(param))) {
    return:
}
```

Reply:
Unfortunately, `some()` checks the presence of at least one allowed param, so urls with both supported and unsupported parameters will pass the validation, so we need to check every single param in the url

Reply:
Oh, I see. Nice catch of the edge case!

I think we can do couple of minor tweaks in the function then:
- name can better fit to a function that returns a boolean
- `allowedParameters` does not to be an argument, it's a global constant
- no need to escape curly brackets
- no need for `isValid` helper variable, we can directly return
- simpler falsy check in the while loop

```js
const isPathValid = (path: string): boolean => {
	const paramsMatcher = RegExp('{(.*?)}', 'g');

	let matches;

	while ((matches = paramsMatcher.exec(path))) {
		if (!ALLOWED_ENDPOINTS_PARAMETERS.includes(matches[1])) {
			return false;
		}
	}

	return true;
};

```
Would this work?